src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx

```diff
@@ -189,7 +190,7 @@ export const View = (props: Props) => {
           isPremiumBrokerRemovalEnabled={props.enabledFeatureFlags.includes(
             "PremiumBrokerRemoval",
           )}
-          isPremiumUser={hasPremium(props.user)}
+          isPremiumUser={isPlusUser}
```

Comment:
I do agree we should change this, but I am not sure we should change this to `plus` from `premium` piecemeal like this since it leads to confusing inconsistencies like this where we are setting `isPremiumUser={isPlusUser}`.

I would suggest filing a ticket in the backlog to replace all `premium` with `plus` in one go as it's own PR, and make this current one much more targeted.